main/nlu/training_data.yml

```diff
@@ -1,5 +1,9 @@
 version: "3.1"
 nlu:
+- lookup: weather_when
```

Comment:
have you tried defining a lookup in a specialised space and can you confirm it works?

additionally, do we know what happens when users write conflicting lookup tables in 2 different spaces? (e.g. same lookup name, different table)